Author: James Westby <james.westby@ubuntu.com>
Description: Do not clear the log file on each operation, but instead append to it.
 This makes debugging of several suspends much easier.
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=25255
Bug-Ubuntu: https://launchpad.net/bugs/410352

Index: pm-utils/pm/pm-functions.in
===================================================================
--- pm-utils.orig/pm/pm-functions.in	2010-02-03 00:34:30.340474179 +0100
+++ pm-utils/pm/pm-functions.in	2010-02-03 17:14:54.032006940 +0100
@@ -269,7 +269,7 @@
 		return 1
 	fi
 	export LOGGING=true
-	exec > "$1" 2>&1
+	exec >> "$1" 2>&1
 }
 
 check_suspend() { [ -n "$SUSPEND_MODULE" ]; }
